orm.go

```diff
@@ -453,3 +469,19 @@ const (
 	TimeQuantumMonthDayHour     TimeQuantum = "MDH"
 	TimeQuantumYearMonthDayHour TimeQuantum = "YMDH"
 )
+
+type CacheType string
+
+// CacheType constants
+const (
+	CacheTypeDefault CacheType = ""
+	CacheTypeLRU     CacheType = "lru"
+	CacheTypeRanked  CacheType = "ranked"
+)
+
+func encodeMap(m map[string]interface{}) string {
+	result, _ := json.Marshal(m)
+	// ignoring the error, since this function is package private
```

Comment:
If this is expected never to generate a non-nil error, we should panic if it does rather than silently ignoring it.